src/main/java/io/cdap/plugin/successfactors/source/service/SuccessFactorsService.java

```diff
@@ -350,23 +356,50 @@ private InputStream filterExpandedEntityData(InputStream dataStream) throws IOEx
     for (JsonNode objectNode : arrayNode) {
       String expandOption = pluginConfig.getExpandOption();
       if (expandOption.contains(SuccessFactorsUrlContainer.PROPERTY_SEPARATOR)) {
-        expandFieldList = Arrays.asList(pluginConfig.getExpandOption()
-                                          .split(SuccessFactorsUrlContainer.PROPERTY_SEPARATOR));
+        expandFieldList = Arrays.asList(expandOption.split(SuccessFactorsUrlContainer.PROPERTY_SEPARATOR));
+      } else if (expandOption.contains(SuccessFactorsUrlContainer.NAV_PROPERTY_SEPARATOR)) {
+        expandFieldList.add(Arrays.asList(expandOption.split(SuccessFactorsUrlContainer.NAV_PROPERTY_SEPARATOR))
+                              .get(0));
+        expandFieldList.add(expandOption.split(SuccessFactorsUrlContainer.NAV_PROPERTY_SEPARATOR)[0]);
```

Comment:
Good catch. While reworking an internal review comment, I missed removing the previous statement. (Line 361)
"a" gets added only once now, in case of expandOption = "a/b/c"
Thanks. 05653e629d98255df503d026be265cd583315a1b